ldcoolp/curation/inspection/readme/__init__.py

```diff
@@ -253,7 +254,16 @@ def retrieve_article_metadata(self):
         # Retrieve description (single string), strip vertical white space
         description = self.article_dict['item']['description'].replace('<div>', '')
         description = description.replace('</div>', '')
-        readme_dict['description'] = html2text(description)
+
+        # Strip ReDATA footer
+        if self.curation_dict['footer'] in description:
```

Comment:
This works. It's funny because I never use `in` for strings.

Reply:
Yea, just a solution to check.